### PR TITLE
Implement Bunyan + Seq logging

### LINKED ISSUE #410: Replace logging with Bunyan + Seq
In an effort to provide a centralised logging solution, the discord bot has move to Seq logging, and the website should too.

https://docs.datalust.co/docs/using-nodejs

### PR DESCRIPTION
**Pull request by TheRealHona**
Closes #410 

Awaiting testing

### CODE REVIEW

File: server/config/express.js [hunk 1]
```diff
@@ -146,7 +167,7 @@ module.exports = (app, config) => {
 		app.use((err, req, res, next) => {
 			const status = err.status || (isCelebrate(err) ? 400 : 500);
 			if (status === 500 || isCelebrate(err))
-				console.error(err);
```
Comment (Gocnak) [inline server/config/express.js:149]: For the test environment, we should probably have this still be console. Tests are only run locally or in CI, not on production.

**Comment by Gocnak** [comment]
I also noticed some discrepancies with spacing, the server uses tabs. When the server linter (#237 ) happens, this can change, but for now stay consistent with tabs.

File: server/config/express.js [hunk 2]
```diff
@@ -17,17 +16,39 @@ const express = require('express'),
...
+			level: 'info'
+			})
+		]
+	})
+	
+	if (app.get('env') === 'development' || app.get('env') === 'production') {
+		app.use(bunyanMiddleware(
+			{
+				logger: logger, 
+			}));
```
Comment (Gocnak) [inline server/config/express.js:51]: This can be one line

File: server/config/express.js [hunk 3]
```diff
@@ -17,17 +16,39 @@ const express = require('express'),
...
+		name: 'Website',
+		streams: [
+			{
+				stream: process.stdout,
+				level: 'warn'
+			},
+			seq.createStream({
+			serverUrl: process.env.SEQ_ADDRESS,
+			apiKey: process.env.SEQ_TOKEN,
+			level: 'info'
```
Comment (Gocnak) [inline server/config/express.js:42]: Indent once more for easier readability

File: server/config/express.js [hunk 4]
```diff
@@ -17,17 +16,36 @@ const express = require('express'),
...
+		streams: [
+			{
+				stream: process.stdout,
+				level: 'warn',
+			},
+			seq.createStream({
+				serverUrl: process.env.SEQ_ADDRESS,
+				apiKey: process.env.SEQ_TOKEN,
+				level: 'info',
+			})
```
Comment (Gocnak) [inline server/config/express.js:43]: Also, what happens if people don't have these defined? Is Seq now a requirement for development or does this gracefully handle it?

File: server/config/express.js [hunk 5]
```diff
@@ -17,17 +16,63 @@ const express = require('express'),
...
 const swaggerSpec = swaggerJSDoc({
 	swaggerDefinition: swaggerDefinition,
 	apis: ['./docs/**/*.yaml'],
 });
 
+function wrappedStdout() {
+	return {
+		write: (entry) => {
+			// Format console output to be human readable
+			var logObject = JSON.parse(entry);
```
Comment (Gocnak) [inline server/config/express.js:33]: Var is pretty bad to use, use `let` or `const`, in this case it's `const` as `logObject` never gets re-assigned to.
```suggestion
			const logObject = JSON.parse(entry);
```

**Comment by TheRealHona** [comment]
Yep, I will rebase this and change to `const`

File: server/config/express.js [hunk 6]
```diff
@@ -17,17 +16,63 @@ const express = require('express'),
...
 	swaggerDefinition: swaggerDefinition,
 	apis: ['./docs/**/*.yaml'],
 });
 
+function wrappedStdout() {
+	return {
+		write: (entry) => {
+			// Format console output to be human readable
+			const logObject = JSON.parse(entry);
+			severity = bunyan.nameFromLevel[logObject.level].toUpperCase();
```
Comment (Gocnak) [inline server/config/express.js:34]: Wait I just noticed this, doesn't this need a `const` as well?

**Pull request merged**